doc/modules/mod_mam.md

```diff
@@ -22,7 +22,7 @@ If you haven't chosen any of the above, skip the next part.
 **Options:**
 
 * `mod_mam_muc`:
-    * `host` (optional, default: `"conference.@HOST@"`): MUC host that will be archived; **Warning:** If you are using MUC Light, make sure this option is set to MUC Light domain.
+    * `muc_module` (optional, default: `mod_muc`): MUC module the MAM service will attach to; **Warning:** If you are using MUC Light, make sure this option is set to `mod_muc_light`.
```

Comment:
Not sure, is it even verified that `mod_mam_muc` can be started 2 times to support both `muc `and `muc_light`? If so, I think this change won't break anything because it just adds a translation step during init from module name to its subhost.